doc/make.bat

```diff
@@ -0,0 +1,49 @@
+@ECHO OFF
+
+pushd %~dp0
+
+REM Command file for Sphinx documentation
+
+if "%SPHINXBUILD%" == "" (
+	set SPHINXBUILD=sphinx-build
+)
+set SOURCEDIR=src
+set BUILDDIR=_build
+set PORT=3000
+set HOST=localhost
+
+if "%1" == "html" goto html
+if "%1" == "help" goto help
+if "%1" == "livehtml" goto livehtml
+
+%SPHINXBUILD% >NUL 2>NUL
+if errorlevel 9009 (
+	echo.
+	echo.The 'sphinx-build' command was not found. Make sure you have Sphinx
+	echo.installed, then set the SPHINXBUILD environment variable to point
+	echo.to the full path of the 'sphinx-build' executable. Alternatively you
+	echo.may add the Sphinx directory to PATH.
+	echo.
+	echo.If you don't have Sphinx installed, grab it from
+	echo.https://www.sphinx-doc.org/
+	exit /b 1
+)
+
+:html
+mkdir %BUILDDIR%\logo
+python ./generate_logos_win32.py -d
+%SPHINXBUILD% -M %1 %SOURCEDIR% %BUILDDIR% %SPHINXOPTS% %O%
```

Comment:
We don't need to copy everything from the Makefile here, but we should at least include the `SPHINXOPTS=-W --keep-going` as otherwise it won't make the build fail on warnings like it should.

Reply:
No, it makes it so that build *does* fail when there are warnings (the `--keep-going` makes it show all warnings instead of just the first one).